doc/xmlConfigFile.md

```diff
@@ -362,3 +362,7 @@ This is useful when the main process is a console, which can respond to Ctrl+C c
 ```xml
 <stopparentprocessfirst>true</stopparentprocessfirst>
 ```
+
+## XML schema validation
+
+We are using embedded [XSD](../src/Core/WinSWCore/XMLSchema.xsd) file to validate XML configurations file schema. You can find sample XSD schema file [here](../examples/windowsservice.xsd). You can follow this schema file when creating the XML configuration file.
```

Comment:
> to validate XML configurations file schema

I don't think this is correct. Are we validating the schema or using the schema to validate XML?

Reply:
Sorry for bad English. Should be change. I'll update.